src/lib/components/primitives/select/select.svelte

```diff
@@ -13,15 +13,34 @@
   export let labelClasses = '';
   export let labelOrder: 'first' | 'last' = 'first';
   export let onChange: (event: Event) => void;
-  export let options: { value: string | null; text: string }[];
+  export let options: { value: string; text: string }[];
 
   let inputEl: HTMLSelectElement;
+  let shouldDisplayOption = true;
+
+  /**
+   * If label and option share the same text, only display option
+  */
+  if (defaultOption === null && (options[0].text === label)) {
+    shouldDisplayOption = false;
+  }
```

Comment:
The comment mostly says it, but it's just visual. This prevents redundant label and option messaging.